zebrad/src/config.rs

```diff
@@ -43,4 +43,8 @@ pub struct ZebradConfig {
     #[serde(skip_serializing_if = "zebra_rpc::config::mining::Config::skip_getblocktemplate")]
     /// Mining configuration
     pub mining: zebra_rpc::config::mining::Config,
+
+    #[cfg(feature = "zebra-scan")]
+    /// Scanner configuration
+    pub scan: zebra_scan::config::Config,
```

Comment:
Note: Using the feature to enable the config is ok for now.

I added a task to the tracking issue to warn when the config is set but the feature is not. (Currently we panic, which is ok for an MVP.)

I also added a task to ticket #7892 to test parsing experimental configs. So we don't have to worry about config tests for the MVP.